libs/wingc/src/lsp/snapshots/completions/struct_literal_value.snap

```diff
@@ -1,64 +1,16 @@
 ---
 source: libs/wingc/src/lsp/completions.rs
 ---
-- label: assert
-  kind: 3
-  detail: "(condition: bool): void"
+- label: "y:"
+  kind: 5
+  detail: num
   documentation:
     kind: markdown
-    value: "```wing\nassert: (condition: bool): void\n```\n---\nAsserts that a condition is true\n\n### Parameters\n- `condition` — The condition to assert"
-  sortText: cc|assert
-  insertText: assert($0)
+    value: "```wing\ny: num\n```"
+  sortText: "ab|y:"
+  insertText: "y: $1"
```

Comment:
This is a regression because we should be showing normal expression completion. This is probably related to the preceding text calculation (in this case, the last character should be ":")

Reply:
Fixed